jsk_rosbag_tools/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ catkin_package(
   CATKIN_DEPENDS
 )
 
-if($ENV{ROS_DISTRO} STREQUAL "noetic")
+if($ENV{ROS_DISTRO} STRGREATER "noetic")
```

Comment:
`STRGREATER` does not include "noetic", need to use `STRGREATER_EQUAL`

Reply:
I think only using `STREQUAL` is better as [the advice](https://github.com/jsk-ros-pkg/jsk_3rdparty/issues/509#issuecomment-2482742076) . It is unpredictable what `ROS_DISTRO` will be in the future release. In my environment, the `ROS_DISTRO` is `one`